brainscore/benchmarks/baker2022.py

```diff
@@ -66,6 +66,10 @@ def __call__(self, candidate: BrainModel):
             raw_score = self._metric(labels, self._assembly)
             ceiling = self._ceiling(self._assembly)
         score = raw_score / ceiling
+
+        # cap score at 1 if ceiled score > 1
+        score[0] = 1 if score[0] > 1 else score[0]
```

Comment:
I advise against positional indexing (with xarray, but also in general) to avoid mis-indexing and make the code easier to parse. E.g. in this case I think you want to do `score.__setitem__({'aggregation': score['aggregation'] == 'center'}, 1)`